.github/label-actions.yml

```diff
@@ -0,0 +1,91 @@
+# Configuration for Label Actions - https://github.com/dessant/label-actions
+
+# When a discussion is resolved, lock the discussion
+'status/resolved':
+  discussions:
+    close: true
+    close-reason: 'resolved'
+    lock: true
+
+# Request more information from the author
+'auto/needs-more-info':
+  comment: >
+    :wave: @{issue-author}, please provide the requested information.
+  issues:
+    label: 
+      - 'status/needs-more-info' 
+  prs:
+    label: 
+      - 'status/needs-more-info' 
+
+# Thank the author for their response, remove auto-labels
+'auto/got-info':
+  comment: >
+    :wave: @{issue-author}, thank you for your response!
+  issues:
+    unlabel: 
+      - 'auto/needs-more-info'
+      - 'status/needs-more-info'
+  prs:
+    unlabel: 
+      - 'auto/needs-more-info'
+      - 'status/needs-more-info'
+
+# Comment that the issue will be transferred to another repository
+'auto/transfer':
+  comment: >
+    :wave: @{issue-author}, this issue will be transferred to another repository.
+  lock: true
+  lock-reason: 'off-topic'
+  issues:
+    label: 
+      - 'needs/transfer'
+      - 'status/wontfix'
```

Comment:
I think this would only matter when transferring issues not part of the org. For issues within the org we can use the "Transfer issue" functionality: https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository

![image](https://github.com/docker/buildx/assets/1951866/0ff0e91c-1be7-409d-8bc8-51c6f3de0ee6)

Reply:
Yep, that was the hope. This set of labels was supposed to convey to the author that the issue was _going_ to be transferred. Likely a silly state and something we can express with `status/`